lib/internal/Magento/Framework/Logger/Test/Unit/Handler/SystemTest.php

```diff
@@ -27,20 +29,22 @@ class SystemTest extends TestCase
     private $filesystemMock;
 
     /**
-     * @var Exception|Mock
+     * @var ExceptionHandler|Mock
      */
     private $exceptionHandlerMock;
 
     /**
      * @inheritdoc
+     *
+     * @throws Exception
      */
     protected function setUp(): void
     {
         $this->filesystemMock = $this->getMockBuilder(DriverInterface::class)
             ->getMockForAbstractClass();
```

Comment:
```suggestion
        $this->filesystemMock = $this->createMock(DriverInterface::class);
```

Reply:
Hello @ihor-sviziev 
Thank you for your notice.
I fixed it.